pkgs/tools/networking/maubot/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, python3
+}:
+let python = python3.override {
+  packageOverrides = self: super: {
+    click = super.click.overridePythonAttrs (oldAttrs: rec {
+        version = "7.1.2";
+        src = oldAttrs.src.override {
+          inherit version;
+          sha256 = "d2b5255c7c6349bc1bd1e59e08cd12acbbd63ce649f2588755783aa94dfb6b1a";
+        };
+    });
+    mautrix = super.mautrix.overridePythonAttrs (oldAttrs: rec {
+      version = "0.12.5";
+      src = oldAttrs.src.override {
+        inherit version;
+        sha256 = "1vm1rsbh2lifa0fsrqxzrq6q0iiww2imrz7v2qs091dawp25q2gp";
+      };
+      propagatedBuildInputs = [ super.aiohttp super.simplejson super.pyramid ];
+    });
+    sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
+      version = "1.3.24";
+      src = oldAttrs.src.override {
+        inherit version;
+        sha256 = "06bmxzssc66cblk1hamskyv5q3xf1nh1py3vi6dka4lkpxy7gfzb";
+      };
+    });
+  };
+};
+in with python.pkgs; python.pkgs.buildPythonApplication rec {
+  pname = "maubot";
+  version = "0.2.0";
```

Comment:
Newest version seems to be 0.2.1

Reply:
0.2.1 is a docker-only release using same code as 0.2.0